src/Tests/Microsoft.NET.TestFramework/ProjectConstruction/TestProject.cs

```diff
@@ -61,6 +65,9 @@ public TestProject([CallerMemberName] string name = null)
 
         public Dictionary<string, string> EmbeddedResources { get; } = new Dictionary<string, string>();
 
+        /// <summary>
+        /// Use this dictionary to set a property key to a desired value before evaluation.
```

Comment:
```suggestion
        /// Use this dictionary to set add properties and values to the created project.
```

Reply:
Suggestion: ` /// Use this dictionary to set add properties and values to the created project.` -->
Accepted Revision: ` /// Use this dictionary to set a property (the key) to a value for the created project.`